lib/metanorma/collection_renderer.rb

```diff
@@ -303,6 +332,15 @@ def update_anchors(bib, docxml, _id) # rubocop:disable Metrics/MethodLength, Met
       end
     end
 
+    def to_ncname(s)
```

Comment:
Can we just reuse the Metanorma Standoc function definition? You can move to_ncname() from metanorma-standoc/lib/asciidoc/cleanup_inline.rb to metanorma-standoc/lib/asciidoc/utils.rb, so that it can be invoked as `::Asciidoctor::Standoc::Utils.to_ncname()`, as a module method.

Reply:
Done, created PR: https://github.com/metanorma/metanorma-standoc/pull/394